themes/classic/templates/checkout/_partials/order-final-summary.tpl

```diff
@@ -72,6 +72,7 @@
          subtotals=$cart.subtotals
          totals=$cart.totals
          labels=$cart.labels
+         is_final_summary=true
```

Comment:
The variable name is not really explicit, can you rename it to `add_product_link`?